CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to AFDKO
+
+First, **thank you** for your interest in contributing! We welcome contributions to all aspects of the project, including Python and C code, tests, and documentation (including questions).
+
+Following are some guidelines and information about contributing, mostly so you know what to expect at various stages in the process.
+
+## Project Goals
+ - Maintain a _high-quality_, useful set of tools for working with OpenType fonts
+ - Keep the toolset current with changes related standards (Unicode, OpenType, etc.)
```

Comment:
changes related `to` standards ?

Reply:
"changes _to_ related standards" is what I meant 😃. Will update.